web/demo/worker.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  createPAGWorker,
+  MAX_ACTIVE_WORKER_CONTEXTS,
+  WorkerPAGFile,
+  WorkerPAGImage,
+  WorkerPAGView,
+} from '../src/worker/client';
+
+window.onload = async () => {
+  const PAG_TOTAL_NUM = Math.min(4, 16);
+  let PAG_NUM_IN_WORKER = Math.min(2, MAX_ACTIVE_WORKER_CONTEXTS);
+
+  const url = './assets/test.pag';
+  const buffer = await fetch(url).then((res) => res.arrayBuffer());
+  const pagFilesPromises = [];
+  let count = 0;
+  let currentPAGWorker: Worker | null = null;
+
+  for (let index = 0; index < PAG_TOTAL_NUM; index++) {
+    if (count % PAG_NUM_IN_WORKER === 0) {
+      currentPAGWorker = await createPAGWorker({
+        locateFile: (file) => '../lib/' + file,
+        workerScriptURL: './libpag.js',
```

Comment:
Please add a comment about libpag.js, how it's generated, where is the configuration file.